README.md

```diff
@@ -7,6 +7,10 @@
 
 This will provide you with a starting Drupal project that is managed with Composer. The install will include a small set of contrib modules, a starting custom module for specific for the build, and a custom starting theme generated from Emulsify.
```

Comment:
"a starting custom module for specific for the build, and a custom starting theme generated from Emulsify." This line needs to be reworded.

Reply:
thanks @birchy82 , fixed that and a couple other little things I noticed. ready for re-review.